packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesUtils.kt

```diff
@@ -277,4 +278,50 @@ object UpdatesUtils {
       throw e
     }
   }
+
+  private const val TOKEN: String = "([a-zA-Z0-9-!#$%&'*+.^_`{|}~]+)"
+  private const val QUOTED: String = "\"([^\"]*)\""
+  private val PARAMETER_PATTERN: Pattern = Pattern.compile(";\\s*(?:\\s*$TOKEN\\s*=\\s*(?:$TOKEN|$QUOTED))?\\s*")
```

Comment:
```suggestion
  private val PARAMETER_PATTERN: Pattern by lazy {
    Pattern.compile(";\\s*(?:\\s*$TOKEN\\s*=\\s*(?:$TOKEN|$QUOTED))?\\s*")
  }
```

nit: maybe lazy loading this property if that's not be used always